src/Confluent.Kafka/Config.cs

```diff
@@ -1,5 +1,3 @@
-// *** Auto-generated *** - do not modify manually.
```

Comment:
`Config_gen` is what now contains the auto-generated classes, `Config` contains a base class used in that.